src/components/loader/loader-square.style.ts

```diff
@@ -23,25 +23,30 @@ const loaderAnimation = keyframes`
 const getDimentions = (size: StyledLoaderSquareProps["size"]) => {
   let width;
   let marginRight;
+  let borderRadius;
 
   switch (size) {
     case "medium":
       width = "16px";
       marginRight = "8px";
+      borderRadius = "12px";
```

Comment:
The value is not dependant on the `size` prop so doesn't need to be added multiple times here

Reply:
I disagree about that one as it takes small and large size loader back to their square shape.